app/routes/company/components/CompaniesPage.js

```diff
@@ -3,65 +3,78 @@
 import React from 'react';
 import styles from './Company.css';
 import LoadingIndicator from 'app/components/LoadingIndicator';
+import InfiniteScroll from 'react-infinite-scroller';
 import { Link } from 'react-router';
 import { Image } from 'app/components/Image';
 import type { Company } from 'app/models';
-import { Button } from 'app/components/Form';
 
 type Props = {
   companies: Array<Company>,
+  fetchMore: () => void,
   showFetchMore: () => void,
-  fetchMore: () => void
+  hasMore: Boolean
 };
 
-export function CompanyItem({ company }: any) {
+const CompanyItem = function({ company }: any) {
+  let websiteString = company.website.split('/');
+  websiteString = websiteString[0] + '//' + websiteString[2];
```

Comment:
instead of this, you can use the new array destructoring, to make the code more readable.

```js
const [protocol, ,domain] = company.website.split('/'); // not the double comma
const websiteString = `${protocol}//${domain}`;
```

Reply:
What do mean `//not the double comma` ? If he wants the desired effect of getting the `https:` and the domain `github.com` he would need the double comma? Right?
<img width="421" alt="screenshot 2018-10-26 at 12 36 49" src="https://user-images.githubusercontent.com/23152018/47561615-189c2c80-d91c-11e8-912e-844a55fd441b.png">

Reply:
yep, thats alot cleaner